### PR TITLE
Vue.js use proper CDN

### LINKED ISSUE #3918: Media Library broken : use proper CDN
https://vuejs.org/js/vue.js is down so is our script link.

```
<script asp-src="https://vuejs.org/js/vue.min.js" debug-src="https://vuejs.org/js/vue.js" asp-name="vuejs" at="Foot"></script>
```

image

### PR DESCRIPTION
**Pull request by Skrypt**
Fixes #3918

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.ContentFields/Views/Media-HtmlField.Modal.cshtml [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 <script asp-src="~/OrchardCore.Media/Scripts/media.js" asp-name="media" at="Foot" depends-on="admin, vuejs, sortable, vuedraggable"></script>
 <style asp-src="~/OrchardCore.Media/Styles/media.min.css" debug-src="~/OrchardCore.Media/Styles/media.css"></style>
-<script asp-src="https://vuejs.org/js/vue.min.js" debug-src="https://vuejs.org/js/vue.js" asp-name="vuejs" at="Foot"></script>
+<script asp-src="https://cdn.jsdelivr.net/npm/vue@2.6.10/dist/vue.min.js" debug-src="https://cdn.jsdelivr.net/npm/vue@2.6.10/dist/vue.js" asp-name="vuejs" at="Foot"></script>
 <script asp-src="https://cdn.jsdelivr.net/sortable/1.4.2/Sortable.min.js" debug-src="https://cdn.jsdelivr.net/sortable/1.4.2/Sortable.js" asp-name="sortable" at="Foot"></script>
 <script asp-src="https://cdnjs.cloudflare.com/ajax/libs/Vue.Draggable/2.14.1/vuedraggable.min.js" debug-src="https://cdnjs.cloudflare.com/ajax/libs/Vue.Draggable/2.14.1/vuedraggable.js" asp-name="vuedraggable" depends-on="vuejs, sortable" at="Foot"></script>
```
Comment (petedavis) [inline src/OrchardCore.Modules/OrchardCore.ContentFields/Views/Media-HtmlField.Modal.cshtml]: Given there is a HTTPS setup cost, maybe we should move Vue.Dragable and vue-multiselect over to jsdeliver also?

Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.ContentFields/Views/Media-HtmlField.Modal.cshtml]: If we do this then maybe we should also then use the same CDN for all our assets.

Comment (petedavis) [reply src/OrchardCore.Modules/OrchardCore.ContentFields/Views/Media-HtmlField.Modal.cshtml]: Yeah that would be ideal. Save establishing so many connections to get all the resources for a page. With HTTP/2 and a single CDN, you should only end up with two connections. One to the web site, and another to the CDN.

jsdeliver looks like a good CDN. I had not seen this one before.

**Comment by jptissot** [comment]
Should we add vue to the OrchardCore.Resources project instead of having these links duplicated ?

**Comment by Skrypt** [comment]
I think we need to merge https://github.com/OrchardCMS/OrchardCore/pull/3928 first

**Pull request merged**